pkg/alertmanager/amcfg_test.go

```diff
@@ -2301,9 +1879,7 @@ templates: []
 			}
 
 			// Verify the generated yaml is as expected
-			if diff := cmp.Diff(tc.expected, string(cfgBytes)); diff != "" {
-				t.Errorf("Unexpected result (-want +got):\n%s", diff)
-			}
+			golden.Assert(t, string(cfgBytes), tc.golden)
```

Comment:
I feel like the diff is really useful sometimes. Not a hard blocker, but it would be nice to keep it somehow if possible :)

Reply:
it seems that gotest.tools/v3/golden uses github.com/google/go-cmp under the hood so it will display the nice diff :)

Reply:
Noice! TIL